cmd/collector/app/span_processor_test.go

```diff
@@ -456,7 +459,14 @@ func TestSpanProcessorCountSpan(t *testing.T) {
 			p.background(10*time.Millisecond, p.updateGauges)
 
 			p.processSpan(&model.Span{}, "")
-			assert.NotEqual(t, uint64(0), p.bytesProcessed)
```

Comment:
this was always a bogus check since it compared pointer with uint64, and it was flagged by the new linter. But then the tests required the conditional check